lib/watir/elements/html_element.rb

```diff
@@ -0,0 +1,7 @@
+module Watir
+  class HTMLElement < Element
+
+    alias_method :attribute, :attribute_value
```

Comment:
`element.rb` needs to be loaded before `html_elements.rb` and the alias needs to happen after `html_elements.rb`. The alternative is renaming to `#set_attribute` which I think I might actually prefer to this.